README.md

```diff
@@ -1,6 +1,8 @@
-# Index Image Build (IIB) Service
+# IIB Image Builder (IIB) Service
 
-A REST API to manage operator index container images
+A REST API to manage operator index container images (and some bundle images).
+
+IIB is a recursive acronym.  IIB stands for IIB Image Builder.
```

Comment:
I would just add a note that it was originally "Index Image Build", otherwise people will start to wonder who came up with such a crazy acronym. :)